core/src/fragment/write_state.cc

```diff
@@ -515,7 +515,7 @@ int WriteState::write_segment(int attribute_id, bool is_var, const void *segment
     } */
 
   // Write_segment directly
-  int rc;
+  int rc = TILEDB_OK;
```

Comment:
Being pedantic here, could you use TILEDB_WS_OK?

Reply:
DONE.